### DISCUSSION TITLE
how to setup Git Hub for my organization and Billing info.

### DISCUSSION THREAD

**Discussion by nikhilghadge1010** [community/community#66858] [New to GitHub]
Dear friends,

We have created a few UI5 applications on SAP BAS (BTP) on the DEV subaccount. We want to now move the changes to QAS (quality) subaccount and eventually to the production subaccount.

I need help to understand how Git Hub can be set up for organizational usage. What is the Billing structure? Is it completely free? if not, how and how many licenses to procure? And what are the best practices to follow while setting up Git Hub?

**Comment by queenofcorgis** [comment]
Hi there 👋🏼 , 

We've got you covered with our docs, but in short we do offer a free experience on Organizations. Take a look below as this should help answer your questions and set you up for success!
* [Types of GitHub Accounts](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#organization-accounts)
* [Creating a new organization from scratch](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)
* [Best Practices for Organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/best-practices-for-organizations)
